docs/docs/Indexing_JSON.md

```diff
@@ -280,6 +299,47 @@ When JSONPath leads to multiple numerical values:
   - `null` values are skipped
   - Any other value type is causing an indexing failure
 
+## Index JSON arrays as VECTOR
+
+Starting with RediSearch 2.6.0, a JSONPath leading to an array of numerical values may be indexed as VECTOR type in the index schema.
+
+If you want to index *multiple* numerical arrays as VECTOR, use a JSONPath leading to multiple numerical arrays using JSONPath operators such as wildcard, filter, union, array slice, and/or recursive descent.
+
+For example, to index a vector embeddings list that correspond images of the same product, specify the JSONPath `$.embeddings[*]` in the schema definition during index creation:
```

Comment:
I think we should add a few more multi-value examples, or link to somewhere we explain JSON dynamic paths

Reply:
Added a link to the JSON Path page in RedisJson